src/codegen/sys/cargo_toml.rs

```diff
@@ -46,15 +46,52 @@ fn fill_empty(root: &mut Table, env: &Env, crate_name: &str) {
 
     let deps = upsert_table(root, "dependencies");
     for ext_lib in &env.config.external_libraries {
+        let dep = upsert_table(deps, &ext_lib.crate_name);
         let ext_package = if ext_lib.crate_name == "cairo" {
             format!("{}-sys-rs", ext_lib.crate_name)
         } else if ext_lib.crate_name == "gdk_pixbuf" {
             "gdk-pixbuf-sys".into()
         } else {
             format!("{}-sys", ext_lib.crate_name)
         };
-        let dep = upsert_table(deps, &*ext_package);
-        set_string(dep, "git", "https://github.com/gtk-rs/gtk-rs-core");
+        let repo_url = match ext_package.as_str() {
+            "cairo-sys-rs" | "gdk-pixbuf-sys" | "gio-sys" | "gobject-sys" | "glib-sys"
+            | "graphene-sys" | "pango-sys" | "pangocairo-sys" => {
+                "https://github.com/gtk-rs/gtk-rs-core"
+            }
+            "atk-sys" | "gdk-sys" | "gdkwayland-sys" | "gdkx11-sys" | "gtk-sys" => {
+                "https://github.com/gtk-rs/gtk3-rs"
+            }
+            "gdk4-wayland-sys" | "gdk4-x11-sys" | "gdk4-sys" | "gsk4-sys" | "gtk4-sys" => {
+                "https://github.com/gtk-rs/gtk4-rs"
+            }
+            "gstreamer-sys"
+            | "gstreamer-app-sys"
+            | "gstreamer-audio-sys"
+            | "gstreamer-base-sys"
+            | "gstreamer-check-sys"
+            | "gstreamer-controller-sys"
+            | "gstreamer-editing-services-sys"
+            | "gstreamer-gl-sys"
+            | "gstreamer-gl/egl-sys"
+            | "gstreamer-gl/wayland-sys"
+            | "gstreamer-gl/x11-sys"
```

Comment:
```suggestion
            | "gstreamer-gl-egl-sys"
            | "gstreamer-gl-wayland-sys"
            | "gstreamer-gl-x11-sys"
```

Or not?

Reply:
Good catch! I missed those but its fixed now